tests/test_mapping_example.py

```diff
@@ -173,22 +173,25 @@ def test_mapping_riv2_drain(
     drainmat = map_active_mod_dflow1d["mf-drn2dflow1d_flux"]
     assert drainmat.shape == (15, 2)
 
-    drainmat.todense() == (
-        [
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [0.0, 0.0],
-            [1.0, 0.0],
-            [0.0, 1.0],
-        ]
-    )
+    assert (
```

Comment:
This one has a nicer error message: https://numpy.org/doc/stable/reference/generated/numpy.testing.assert_array_equal.html

You can get a ndarray by using toarray instead of todense

Reply:
done